src/test/regress/expected/multi_subquery_in_where_reference_clause.out

```diff
@@ -87,7 +87,7 @@ DETAIL:  Functions are not allowed in FROM clause when the query has subqueries
 SELECT
   user_id
 FROM
-  (SELECT  5 AS user_id) users_reference_table
+  (SELECT  5 AS user_id UNION ALL SELECT 6) users_reference_table
```

Comment:
pg12 has gotten a lot better at optimizing basic `select x` queries, so that without these complication tactics we end up with changed subquery/task ids & constant propagation

Reply:
I'm concerned about this change because it changes the query in a way that Citus behaves (e.g., plans) differently. So, we're not testing the same thing anymore from Citus' perspective. 

Can we find another way of having a subquery without FROM clause? From example, does defining function which always returns 5, and calling it help?

I'm happy to remove it now and leave it pg12 change btw.